update_appdata.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# replace AppID in appdata file using '|' as sed delimiter
+sed -i 's|<id>org.mavlink.qgroundcontrol</id>|<id>org.mavlink.QGroundControl</id>|g' $1
+# update licence
+sed -i 's|<metadata_license>GPL-3.0 AND Apache-2.0</metadata_license>|<metadata_license>CC0-1.0</metadata_license>|g' $1
+# add content_rating and releases
+sed -i -r "s|</description>|</description>\n  <content_rating type=\"oars-1.1\"/>\n  <releases><release date=\"$(date +%F)\" version=\"\"/></releases>|g" $1
```

Comment:
this should be patch. Also file an issue with upstream to fix the lack of content_rating and release.

Reply:
Also this break the idea that the build is reproductible since the change isn't a fixed date.

Reply:
> this should be patch. Also file an issue with upstream to fix the lack of content_rating and release.

There is a pending PR here: https://github.com/mavlink/qgroundcontrol/pull/10615

Reply:
STILL, a FIXED DATE. Since it is a FIXED COMMIT.